### ISSUE TITLE
Some Code Won't Update

### ISSUE THREAD

**Issue by ThePyromancer23** [renpy/renpy#1793]
I am fairly new to Python and am reworking my characters with objects and classes as I have learned how. However, my menu displaying some character stats is not changing to fit the updated code, and in-game, still responds to updates from old variables that aren't even apart of the code anymore.
```
screen love_screen():
    frame:
        align(0.05,0.05)
        xysize(180, 500)
        vbox:
            align(0.0, 0.5)
            text "{u}Relations:{/u}"
            null height 10
            text "[sfu.name_known]: [sfu.gauge_display]"
            null height 10
            text "[nfu.name_known]: [nfu.gauge_display]"
            null height 10
            text "[gfu.name_known]: [gfu.gauge_display]"
            null height 10
            text "[rfu.name_known]: [rfu.gauge_display]"
            null height 10
            text "[m_mm]: [mm_d]"
```

This is the screen that should show names of characters and their display. The default for the name_known is "??". The screen still shows the old default. 

The screen updates with a name when I do:
`$m_nf = "Andre"`
Which is the old variable for changing one of the character names.

tldr; Renpy isn't registering my code changes no matter how many times I save that script.

**Comment by ThePyromancer23** [comment]
Sorry, the code section is a mess, I have never used git hub before and don't know how to get it into lines.

**Comment by renpytom** [comment]
I updated the formattiing, but my guess is the old screen is coming from somewhere. It's possible you deleted a .rpy file but not the corresponding .rpyc file - clicking the Compile Script button in the launcher might fix this for you.

**Comment by ThePyromancer23** [comment]
Ahh, thank you. I'd been doing a lot of experimentation the best way to organize and renamed that script to a broader category. The .rypc was still up and kicking.
